dev-docs/modules/blueconicRtdProvider.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: page_v2
+title: BlueConic Real Time Data Provider
+display_name: BlueConic Real-time Module
+description: BlueConic Real-time Data Module
+page_type: module
+module_type: rtd
+module_code : blueconicRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# BlueConic Real-time Data Submodule
+{:.no_toc}
+
+* TOC
+{:toc}
+
+BlueConic is the Customer Data Platform that liberates customer data to make individualized marketing a reality. We are the first and only company that delivers to marketers the promise of a single view of a customer. BlueConi gather rela-time and batch data of customers from all the channels to the tailor products, services and experiences according to their wants and needs.
+
+The BlueConic real-time data module in Prebid has been built so that publishers
+can maximize the power of their first-party audiences, user-level and contextual data.
+This module provides both an integrated BlueConic identity with real-time
+contextual and audience segmentation solution that seamlessly and easily
+integrates into your existing Prebid deployment. For more information, 
```

Comment:
@Soni-Goyal could you tone the language a little bit down? This is too much marketing for a technical documentation